tests/test-cases/env.yaml

```diff
@@ -15,8 +15,8 @@ tests:
     expect:
       diff: []
       stdout:
-        - 'atmos_base_path = "./"'
-        - 'atmos_cli_config_path = ".*tests.*fixtures.*scenarios.*env.*atmos"'
+        - 'atmos_base_path = ".*tests.*fixtures.*scenarios.*env"'
```

Comment:
I struggle to know what I'm approving here, since this is a very broad regex, that didn't catch the original issue (hence the feature was released prematurely).

Reply:
Please add a comment above this line with an example of the expected value.